ansible/module_utils/port_utils.py

```diff
@@ -112,4 +138,3 @@ def get_port_alias_to_name_map(hwsku):
             port_alias_to_name_map["Ethernet%d" % i] = "Ethernet%d" % i
 
     return port_alias_to_name_map
-
```

Comment:
Line break at the eof removed.

Reply:
Line break is still there, this was an extra newline. :)